src/intervals/interval_arithmetic.jl

```diff
@@ -1,16 +1,25 @@
 
 ## Empty interval:
 
-emptyinterval(T::Type) = Interval(convert(T, NaN))  # interval from Inf to Inf
-emptyinterval(x::Interval) = Interval(oftype(x.lo, NaN))
+@doc doc"""Empty intervals are represented as intervals of `NaN`s.
+The automatic propagation of `NaN`s means that any operation with an empty interval gives back
+an empty interval.""" ->
+
+emptyinterval(T::Type) = Interval(convert(T, NaN))
+emptyinterval(x::Interval) = Interval(convert(eltype(x), NaN))
+
+∅ = emptyinterval(Float64)
```

Comment:
I think it is a good idea to define `∅` as a `const` which depends of the type of intervals that are being used. Something like the way `pi` is treated